sdjournal/journal_test.go

```diff
@@ -204,10 +204,7 @@ func TestJournalGetEntry(t *testing.T) {
 		t.Fatalf("Error writing to journal: %s", err)
 	}
 
-	r := j.Wait(time.Duration(1) * time.Second)
-	if r < 0 {
-		t.Fatalf("Error waiting to journal")
-	}
+	time.Sleep(time.Duration(1) * time.Second)
```

Comment:
Can you expand on why a hard sleep is better than waiting for the journal? This seems counter-intuitive to me as `time.Sleep` cries for flakes later on.